test/screenshot/snackbar/index.scss

```diff
@@ -0,0 +1,6 @@
+.snackbar-container {
+  margin-bottom: 16px;
+  .mdc-snackbar {
+    position: static !important;
```

Comment:
I know this is just a screenshot tests, but the `!important` is a code smell. Why is this needed?

Reply:
By default, snack bar uses position: fixed and second item is displayed over the first one. I have less bad solution for this

Reply:
ah...sounds like an mdc web issue then?

Reply:
no, it is correct behavior for the browser. But for screenshot tests it will not allow to test snackbar correctly. `.mdc-snackbar.mdc-snackbar` can be used instead of !important